packages/terra-aggregator/src/Aggregator.jsx

```diff
@@ -0,0 +1,229 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  /**
+   * The components that will be managed by the Aggregator. Each component provided will be provided with an 'aggregatorDelegate' prop
+   * containing the Aggregator API. Keys must also be provided to allow the Aggregator to uniquely identify the component.
+   * Immutable objects are preferred.
+   */
+  items: PropTypes.arrayOf(PropTypes.shape({
+    key: PropTypes.string.isRequired,
+    component: PropTypes.element.isRequired,
+  })),
+  /**
+   * A function that will override the Aggregators default render. The function will receive an Object parameter data necessary for the
+   * render process.
+   */
+  render: PropTypes.func,
+  /**
+   * A function that will be provided to Aggregator items that have received focus. The function must adhere to the standardized
+   * AppDelegate disclosure API.
+   */
+  disclose: PropTypes.func,
```

Comment:
Why isn't this required?

Reply:
Because the focus management works without it. Enforcement of the disclose prop presence could be managed by a higher component.